src/store-tree/index.ls

```diff
@@ -71,6 +71,12 @@ class StoreTree extends StoreNode
     obj
 
 
+  $debug: ->
+    obj = {}
+    @_for-each-subnode (subnode, key) -> obj[key] = subnode.$debug!
```

Comment:
Unfortunately I don't think this works because it needs to call `$debug` on `@[key]` not `@_children[key]` which is why I created the helper method